src/index.js

```diff
@@ -2,52 +2,140 @@ import './style.css';
 import kebabMenu from './kebab-menu.svg';
 import enterIcon from './enter-icon.svg';
 import refreshIcon from './Refresh_icon.svg';
+import {
+  addToDo,
+  getToDos,
+  removeToDo,
+  updateToDoCompleted,
+  updateToDoListDescription,
+} from './store.js';
 
-const toDoLists = [
-  {
-    description: 'Have my breakfast',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Take my lunch',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Visit my mum',
-    completed: false,
-    index: 3,
-  },
-];
+const form = document.querySelector('.input-form');
```

Comment:
- [ ] Nice job on the add and remove but I did encounter a minor issue in your code that is the input is not working.
please fix it

Reply:
Thank you for your time.
Can you please point to the exact issue because I can't seem to understand what you mean by the input not working?
Thank you.